mitmproxy/net/udp.py

```diff
@@ -0,0 +1,328 @@
+from __future__ import annotations
+
+import asyncio
+from asyncio import base_events
+from collections import deque
+import ipaddress
+import socket
+import struct
+from typing import Any, Deque, Dict, Optional, Tuple, Union
+from mitmproxy import ctx
+
+
+Address = Union[Tuple[str, int], Tuple[str, int, int, int]]
```

Comment:
I had issues using the definition without the 4-tuple IPv6 part. I'll check again, if the issues persist I'll rename the definition to `SockAddress`, so that it doesn't get confused/gets clear why `Address` isn't used.

Reply:
Fixed in 079a59039de5910f88031eee0045be1819e0b474, used `Address` everywhere except in `TransparentSocket` where mypy complained.